utils/interfaces/_logs/core.py

```diff
@@ -154,6 +164,12 @@ def _clean_line(self, line):
 
         return line
 
+    def _get_standardized_level(self, level):
+        if context.library == "php":
+            return level.upper()
+        else:
+            super(_LibraryStdout, self)._get_standardized_level(level)
```

Comment:
The cause of the error in java 

```suggestion
            return super(_LibraryStdout, self)._get_standardized_level(level)
```